README.md

```diff
@@ -1,8 +1,15 @@
 # Awesome Vuetify
 [![Awesome](https://awesome.re/badge.svg)](https://awesome.re)
 
-Below is a list of curated resources for Vuetify. If you have something that you think belongs here, feel free to reach out to us in the [community](#community).
+[<img src="logo.png" align="right" width="101">](https://vuetifyjs.com)
```

Comment:
This appears to be broken.

Reply:
Not exactly sure what's broken, it looks ok on my side (I'm using Chrome on Chrome OS)
![screenshot 2018-11-21 at 23 39 09](https://user-images.githubusercontent.com/123800/48851784-d49b2b00-ede6-11e8-9e6c-4786f165a398.png)

According to the guidelines the logo can be "Either centered, fullwidth, or placed at the top-right", so if the right-alignment is causing issues I could move it.